partner_compassion/models/partner_compassion.py

```diff
@@ -318,6 +325,12 @@ def _compute_criminal_record_name(self):
                     partner.with_context(bin_size=False).criminal_record)
                 partner.criminal_record_name = f"Criminal record {partner.name}{ftype}"
 
+    @api.multi
+    @api.depends("sponsorship_ids")
+    def _compute_write_and_pray(self):
+        for partner in self:
+            partner.write_and_pray = any([t == "SWP" for t in partner.mapped("sponsorship_ids.type")])
```

Comment:
```suggestion
            partner.write_and_pray = "SWP" in partner.mapped("sponsorship_ids.type")
```

Reply:
Thank you, I knew this could be simplified